app/helpers/application_helper/button/physical_storage_timeline.rb

```diff
@@ -0,0 +1,14 @@
+class ApplicationHelper::Button::PhysicalStorageTimeline < ApplicationHelper::Button::EmsTimeline
+  def calculate_properties
+    super
+    self[:hidden] = true unless visible?
+  end
+
+  def visible?
+    @record.supports?(:timeline)
+  end
```

Comment:
These are identical to the parent class, why override them?

Reply:
This file was a reminder from the first commit, it is deleted.